pkg/commands/os.go

```diff
@@ -83,6 +83,27 @@ func (c *OSCommand) ExecutableFromString(commandStr string) *exec.Cmd {
 	return c.command(splitCmd[0], splitCmd[1:]...)
 }
 
+func (c *OSCommand) NewShellCmd(commandStr string) *exec.Cmd {
+	quotedCommand := ""
+	// Windows does not seem to like quotes around the command
+	if c.Platform.os == "windows" {
+		quotedCommand = strings.NewReplacer(
+			"^", "^^",
+			"&", "^&",
+			"|", "^|",
+			"<", "^<",
+			">", "^>",
+			"%", "^%",
+		).Replace(commandStr)
+	} else {
+		quotedCommand = c.Quote(commandStr)
+	}
```

Comment:
I copied this over from `lazygit`. Was there any reason this was not here before?

Reply:
Also, i didn't wrap the shell directly on `ExecutableFromString` because it was touching a lot of different functions and caused some issues. E.g. running `EditFile(filename string) (*exec.Cmd, error)` with it.

Reply:
No good reason. Also be sure to update the Quote method based on lazygit i.e.:
```go
func (self *CmdObjBuilder) Quote(message string) string {
	var quote string
	if self.platform.OS == "windows" {
		quote = `\"`
		message = strings.NewReplacer(
			`"`, `"'"'"`,
			`\"`, `\\"`,
		).Replace(message)
	} else {
		quote = `"`
		message = strings.NewReplacer(
			`\`, `\\`,
			`"`, `\"`,
			`$`, `\$`,
			"`", "\\`",
		).Replace(message)
	}
	return quote + message + quote
}
```